### PR TITLE
LPS-157661 Account avatar not saved in the creation

### PR DESCRIPTION
**Pull request by sontruongces**
Hi team,
This is the fix for [LPS-157661](https://issues.liferay.com/browse/LPS-157661), the original PR has passed my review at sontruongces#19. Please help me review it and give me feedback. Thanks!
cc: @locpham97

### CODE REVIEW

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/AddAccountUserMVCActionCommand.java [hunk 1]
```diff
@@ -83,6 +86,16 @@ protected void doProcessAction(
...
 
+		byte[] portraitBytes = null;
+
+		long fileEntryId = ParamUtil.getLong(actionRequest, "fileEntryId");
+
+		if (fileEntryId > 0) {
+			FileEntry fileEntry = _dlAppLocalService.getFileEntry(fileEntryId);
+
+			portraitBytes = FileUtil.getBytes(fileEntry.getContentStream());
+		}
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/AddAccountUserMVCActionCommand.java:97]: Move this block to just before it's used on line 133. The `portraitBytes` variable is not needed to create the user, so it should not be introduced until after the user is created. (Brian calls this the "as used" SF rule).

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/AddAccountUserMVCActionCommand.java [hunk 2]
```diff
@@ -164,12 +182,19 @@ protected void doProcessAction(
 		}
 	}
 
+	@Reference(unbind = "-")
+	protected void setDLAppLocalService(DLAppLocalService dlAppLocalService) {
+		_dlAppLocalService = dlAppLocalService;
+	}
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/AddAccountUserMVCActionCommand.java:188]: If possible, we should use the `@Reference` annotation directly on the field.

**Review by drewbrokke — CHANGES_REQUESTED** [review]

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/AddAccountUserMVCActionCommand.java (hunk 1 above)
Comment (sontruongces) [reply modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/AddAccountUserMVCActionCommand.java]: @drewbrokke I updated the fix. Thanks!

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/AddAccountUserMVCActionCommand.java (hunk 2 above)
Comment (sontruongces) [reply modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/AddAccountUserMVCActionCommand.java]: I updated the code. Thanks!

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 3]
```diff
@@ -609,11 +613,17 @@ public Map<String, Object> getSystemModelAttributes(
...
+
+			if(Objects.equals(objectField.getDBType(), ObjectFieldConstants.DB_TYPE_STRING)) {
+				value  = LocalizationUtil.getLocalization((String) value, null, true);
+			}
+
 			modelAttributes.put(
 				objectField.getName(),
-				GetterUtil.getObject(
-					baseModelAttributes.get(objectField.getDBColumnName()),
-					objectEntryId));
```
Comment (drewbrokke) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:616]: @sontruongces was this file accidentally committed? It does not seem related.

**Review by drewbrokke — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java (hunk 3 above)
Comment (sontruongces) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: @drewbrokke, Sorry, my bad. I removed the file. Thanks!

**Review by drewbrokke — APPROVED** [review]

**Comment by epark18** [comment]
ci:forward

**Pull request closed without merging**